docs/upgrade-marshmallow.rst

```diff
@@ -0,0 +1,59 @@
+Marshmallow migration to version 3
+==================================
+
+Marshmallow
+-----------
+
+Marhsmallow is a package for serialization and deserialization of complex data structures from and to Python types. You can discover it's usage within Invenio `here <https://invenio.readthedocs.io/en/latest/tutorials/understanding-data-models.html?highlight=marshmallow#define-a-marshmallow-schema>`_
```

Comment:
The guide mainly talks about how to upgrade to v3. I think there's a part missing about what are their options right now:

Options
1) I have Marshmallow v2 schemas
1.1) -> Stay with Marshmallow 2 for now by pining marhsmallow in their instance.
1.2) -> Upgrade to Marshmallow 3. Follow the guide below